### PR TITLE
Support custom ajv options

### PR DESCRIPTION
**Pull request by lisuizhe**
PR against this feature request: https://github.com/apideck-libraries/reva/issues/7

### CODE REVIEW

File: src/Reva.ts [hunk 1]
```diff
@@ -57,6 +64,18 @@ export class Reva {
 
   constructor(options: Partial<RevaOptions> = {}) {
     this.options = { ...this.options, ...options };
+    if (options.paramAjvOptions) {
```
Comment (elsmr) [inline src/Reva.ts]: No need for this `if` and the one on line 73, `...(null|undefined)` is fine

File: src/Reva.ts [hunk 2]
```diff
@@ -42,13 +46,16 @@ export interface RevaValidateOptions {
...
+export const revaDefaultBodyAjvOptions = { allErrors: true, strict: false };
+
 export class Reva {
-  private paramAjv = new Ajv({
-    allErrors: true,
-    coerceTypes: true,
-    strict: false,
-  });
-  private bodyAjv = new Ajv({ allErrors: true, strict: false });
+  private paramAjv = new Ajv(revaDefaultParamAjvOptions);
```
Comment (elsmr) [inline src/Reva.ts]: Let's change this line to `private bodyAjv: Ajv` so we don't have to call `new Ajv` twice. (same for line 58)

File: README.md [hunk 3]
```diff
@@ -85,6 +85,8 @@ Reva is the main **Re**quest **va**lidation class. You can optionally pass optio
   - `allowAdditionalParameters?: true | OpenApiParameterType[]` Allow additional parameters to be passed that are not defined in the OpenAPI operation. Use `true` to allow all parameter types to have additional parameters. Default value: `['header', 'cookie']`
   - `partialBody?: boolean` Ignore required properties on the requestBody. This option is useful for update endpoints where a subset of required properties is allowed. Default value: `false`
   - `groupedParameters?: OpenApiParameterType[]` Validate multiple OpenAPI parameter types as one schema. This is useful for APIs where parameters (`query`,`path`, etc) are combined into a single `parameters` object. Default value: `[]`
+  - `paramAjvOptions?: AjvOptions]` Custom Ajv options for request param validation. Use `true` to allow all parameter types to have additional parameters. Default value: `{ allErrors: true, coerceTypes: true, strict: false `
```
Comment (elsmr) [inline README.md]:
```suggestion
  - `paramAjvOptions?: AjvOptions` Custom AJV options for request parameter validation.
```

File: README.md [hunk 4]
```diff
@@ -85,6 +85,8 @@ Reva is the main **Re**quest **va**lidation class. You can optionally pass optio
...
+  - `bodyAjvOptions?: AjvOptions]` Custom Ajv options for request body validation. Use `true` to allow all parameter types to have additional parameters. Default value: `{ allErrors: true, strict: false `
```
Comment (elsmr) [inline README.md]:
```suggestion
  - `bodyAjvOptions?: AjvOptions` Custom AJV options for request body validation.
```

**Review by elsmr — CHANGES_REQUESTED** [review]
Thanks for your contribution! just some minor remarks to fix, then we can get this merged and released!

File: src/Reva.ts (hunk 1 above)
Comment (lisuizhe) [reply src/Reva.ts]: Thanks for checking & feedback.
Updated.

File: src/Reva.ts (hunk 2 above)
Comment (lisuizhe) [reply src/Reva.ts]: Thanks for checking & feedback.
Updated.

File: README.md (hunk 3 above)
Comment (lisuizhe) [reply README.md]: Thanks for checking & feedback.
Updated.

File: README.md (hunk 4 above)
Comment (lisuizhe) [reply README.md]: Thanks for checking & feedback.
Updated.

**Review by elsmr — APPROVED** [review]
Thanks for following up 🙌

**Comment by lisuizhe** [comment]
> Thanks for following up 🙌

😊Thanks

**Pull request merged**